src/app/frontend/podlist/podcard_component.js

```diff
@@ -82,7 +82,24 @@ export class PodCardController {
    * @export
    */
   isFailed() {
-    return this.pod.podStatus.podPhase === 'Failed' || this.hasWarnings();
+    if (this.pod.podStatus.podPhase === 'Failed' || this.hasWarnings()) {
```

Comment:
What about Succeeded state? should we show an error if the pod is Succeeded but has warnings somehow?